packages/screens/Marketplace/MarketplaceScreen.tsx

```diff
@@ -113,6 +113,8 @@ export const MarketplaceScreen: ScreenFC<"Marketplace"> = () => {
           filterText={filterText}
         />
       </View>
+
+      <SpacerColumn size={16} />
```

Comment:
maybe we should use space_x2 instead of hardcoded value ?

Reply:
`size` value is already multiplied by 8, as `space_xX` value.
We could refactor this, but not in this PR.
Before `SpacerColumn`, there was a big margin, used to add vertical space under the "tables with no element at the bottom"